components/nav.vue

```diff
@@ -53,10 +59,19 @@
         components: {
             Twitter,
             Discourse,
+            Bars,
             StatusIndicator,
         },
         methods: {
             utm,
+            openMenu () {
+                const navBar = this.$refs.nav;
+                if (navBar.className === 'nav') {
+                    navBar.className += ' responsive';
+                } else {
+                    navBar.className = 'nav';
+                }
```

Comment:
```suggestion
            toggleMenu () {
                this.$refs.nav.classList.toggle('responsive');
```
I think we can just toggle the class here?